packages/core/admin/admin/src/content-manager/components/CollectionTypeFormWrapper/index.js

```diff
@@ -186,6 +187,7 @@ const CollectionTypeFormWrapper = ({ allLayoutData, children, slug, id, origin }
     return () => {
       source.cancel('Operation canceled by the user.');
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please don't disable this it causes more problems down the line when people edit the function but don't change their deps, if you can't safely put the fetchclient as a dep you need to change the code in the hook to reflect this.